docs/index.md

```diff
@@ -19,6 +19,10 @@ To get started with an installation, refer to the [roadmap](https://wiki.t2linux
 
 You may contact us on the [Linux on T2 Macs (2018+) Discord Server](https://discord.com/invite/68MRhQu), where you can find most of the people involved.
 
+# Contribute
+
+Visit [this page](https://wiki.t2linux.org/donations/) for more details.
```

Comment:
Can you reword this to "Visit the donations page for more details."? It's better to be transparent about where a link leads to before clicking

Reply:
db0f6f2